src/ngx_http_lua_ssl_certby.c

```diff
@@ -298,8 +298,14 @@ ngx_http_lua_ssl_cert_handler(ngx_ssl_conn_t *ssl_conn, void *data)
 
     c->log->action = "loading SSL certificate by lua";
 
-    rc = lscf->srv.ssl_cert_handler(r, lscf, L);
-
+    if (lscf->srv.ssl_cert_handler) {
```

Comment:
This pointer can never be NULL when it is already in this `ngx_http_lua_ssl_cert_handler` function. And we never observe NULL pointers in some user company's global SSL gateway network. I guess it's something else, like some 3rd-party NGINX C modules or 3rd-party Lua C or FFI libraries you are using, which corrupt memory. This does not look like a culprit. Better dig things up instead of working around the real issue which might be somewhere else.